src/shrutiC/Calculator.java

```diff
@@ -0,0 +1,53 @@
+package shrutiC;
+
+class Calculator {
+
+	void printEvenNos(int startIndex, int endIndex) {
+
+		System.out.println("\nEven numbers between " + startIndex + " and " + endIndex + " are :-");
+		for (int i = startIndex; i <= endIndex; i++) {
+			System.out.println(i);
+			i++;
```

Comment:
what is I give number range from 11, then your code will give wrong output.

if(i%2==0) should placed